src/components/toggle/toggle.stories.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
-
-import { storiesOf } from '@storybook/react'
 import Toggle from './toggle'
-storiesOf('Elements | Toggle', module)
-  .add('active', () => <Toggle active={true} />)
-  .add('inactive', () => <Toggle active={false} />)
-  .add('darkmode', () => <Toggle darkMode={true} />)
+
+export default {
+  title: 'Components/Toggle',
+  component: Toggle,
+};
+
+export const Active = () => <Toggle active={true} />
+export const Inactive = () => <Toggle active={false} />
+export const Darkmode = () => <Toggle darkMode={true} />
+export const DarkmodeActive = () => <Toggle darkMode={true} active={true} />
```

Comment:
This got me thinking, let's chat at some point about how we want to implement dark mode. I think using a prop might not be how we want to go about it.

Reply:
Sounds good. I agree that a prop might not be the right approach.